deeptime/decomposition/_koopman.py

```diff
@@ -670,41 +675,105 @@ def feature_component_correlation(self):
         feature_sigma = np.sqrt(np.diag(self.cov_00))
         return np.dot(self.cov_00, self.singular_vectors_left[:, : self.output_dimension]) / feature_sigma[:, None]
 
+    def ck_test(self, models, test_model=None, include_lag0=True, n_observables=None,
+                observables='phi', statistics='psi', progress=None):
```

Comment:
Not sure if I understand the defaults `phi` and `psi` here?

Reply:
yeah they're not exactly descriptive... although it is explained in the docstring. basically refers to left and right singular vectors